README.md

```diff
@@ -1,72 +1,55 @@
-| `tslint.json`   | Default [TSLint](https://palantir.github.io/tslint/) configuration for apps in the workspace. |
-| `README.md`  | Introductory documentation for the root app. |
 
-## Configuration
+## General info
 
-The configuration provided by `angular.json` file.
+Frontend project for Open360. The Open360 is a system to create, manage and run surveys in a convenient way for both employees and employers.
 
-## Environment Configuration
+### Related project
+[Open360 Backend](https://github.com/o360/backend)
 
-The environment configuration files in `/src/environments/` provide a way for you to set and override configuration settings based on a given environment.
-The `/tools/env/env.config.ts` configuration is set up in base `LOCAL` environment, whereas the `/src/deploy-configs/dev.json` is specific the development environment,
- as is `/src/deploy-configs/prod.json` specific to the prod environment.
-
-## Development environment settings and general information of project structure  
-
-All steps are described for Ubuntu 18.04 LTS. Can be different in other systems.
 
+# Local Development
 ## Prerequisites
 
-Node.js - v10.x and yarn
+Node.js - v12.x and yarn
 
 * ```sudo curl -sL https://deb.nodesource.com/setup_10.x | sudo bash -```
 * ```sudo apt-get update```
 * ```sudo apt-get install nodejs```
 * ```sudo npm install -g npm```
 * ```sudo npm install -g yarn```
 
-[JetBrains WebStorm](https://www.jetbrains.com/webstorm/download/) - latest
-
-### Setting up
-Import IDE's Code Style settings.
+## Installation
 
-* File > Settings > Editor > Code Style > HTML > 
-Do not indent children of: Set only HTML
-* File > Settings > Editor > Code Style > TypeScript > Spaces 
-Set: Object literal braces, ES6 Import/Export braces
-* File > Settings > Editor > Code Style > TypeScript > Other
-Set: Quote marks: Single Quote, use ‘public’ modifier
-
-Allow EditorConfig to overwrite code style settings.
- 
-### Configure TypeScript support:
+To get started locally, follow these instructions:
 
-* File > Settings > Languages & Frameworks > TypeScript
-Disable TypeScript compiler.
-* File > Settings > Languages & Frameworks > TypeScript > TSLint
-Set: Enable. TSLint package will be available after packages install. 
-Set: Search for tslint.json.
+1. If you haven't done it already, [make a fork of this repo](https://github.com/o360/frontend/fork).
+1. Clone to your local computer using `git`.
+1. Make sure that you have Node 12 or later installed. See instructions [here](https://nodejs.org/en/download/).
+1. Make sure that you have `yarn` installed; see instructions [here](https://yarnpkg.com/lang/en/docs/install/).
+1. Run `yarn` (no arguments) from the root of your clone of this project to install dependencies.
 
-## Install packages:
-  `yarn install`
+## Environment Configuration
```

Comment:
IMO this section should be reworked entirely.

Actual config files should be removed and config example should be added instead. And finally example file should be described in the section, option by option.

Reply:
You're right, it will be done in #44